portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase

```diff
@@ -29,6 +29,77 @@ definition {
 		}
 	}
 
+	@description = "LPS-180899 Confirm that when adding several views, pagination is displayed correctly when placing as 4 items"
+	@priority = 5
+	test CanAssertPagination {
+		task ("When The user goes to add a new Dataset") {
+			FrontendDataSetAdmin.createDataSet(
+				key_name = "Test Dataset",
+				key_type = "Channel");
+		}
+
+		task ("And Add a 6 views") {
+			FrontendDataSetAdmin.createDataSetView(
+				dataSetName = "Test Dataset",
+				description = "Description Test",
+				key_name = "View Test");
+
+			FrontendDataSetAdmin.addDataSetView(
+				description = "Description Test",
+				key_name = "View Test");
+
+			FrontendDataSetAdmin.addDataSetView(
+				description = "Description Test",
+				key_name = "View Test");
+
+			FrontendDataSetAdmin.addDataSetView(
+				description = "Description Test",
+				key_name = "View Test");
+
+			FrontendDataSetAdmin.addDataSetView(
+				description = "Description Test",
+				key_name = "View Test");
+
+			FrontendDataSetAdmin.addDataSetView(
+				description = "Description Test",
+				key_name = "View Test");
+		}
+
+		task ("And Change the pagination to 4 items") {
+			FrontendDataSetAdmin.changePagination(itemsPerPage = "4 Items");
+		}
+
+		task ("Then Confirm that 2 pages are shown") {
+			AssertElementPresent(locator1 = "Pagination#NEXT_LINK");
+
+			for (var num : list "1,2,3,4") {
+				AssertElementPresent(
+					key_itemName = "Test Dataset ${num}",
+					locator1 = "FrontendDataSetAdmin#VIEW_PAGINATION");
```

Comment:
```suggestion
					entryName = "View Test ${num}",
					locator1 = "FrontendDataSetAdmin#DATA_SET_VIEW_ENTRY_NAME");
```

Reply:
the views must have different names, the ones you created have the same name and you can use DATA_SET_VIEW_ENTRY_NAME path here and delete VIEW_PAGINATION path